src/modules/auth/controllers/auth.controller.ts

```diff
@@ -0,0 +1,55 @@
+import { Body, Controller, Post, Delete } from '@nestjs/common';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { SWAGGER_TAGS } from 'src/config/swagger/tags';
+import { AuthService } from '../services/auth.service';
+import {
+  SignUpUserBody,
+  UserCredentials,
+  UserRemovalBody,
+} from '../dto/auth.dto';
+
+@ApiTags(SWAGGER_TAGS.AUTH)
+@Controller()
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Post('/login')
```

Comment:
Add docstrings for each function.
You can generate these by selecting the entire function and ask Tabnine to do the magic.